schemas/preferences.sublime-settings.json

```diff
@@ -406,19 +406,60 @@
           "default": true
         },
         "draw_white_space": {
-          "markdownDescription": "Controls when white space is drawn. Any of the valid options may be combined.\nFor example [\"selection\", \"trailing\", \"isolated\"] will draw whitespace at the end of any lines and under the selection.",
+          "markdownDescription": "Controls when white space is drawn.\n\nAny of the following options may be combined:\n\n - `\"selection\"`: Draw white space under the current selection.\n - `\"leading\"`: Draw any white space between the beginning of a line and the first character.\n - `\"enclosed\"`: Draw white space enclosed by other characters.\n - `\"trailing\"`: Draw white space following the last character on a line.\n - `\"isolated\"`: Draw white space on lines containing no other characters.\n - `\"all\"`: All of the above, ie. always draw white space.\n\nThese options may be further refined by appending any of the following separated by an underscore:\n\n - `\"none\"`: Don't draw this kind of white space.\n - `\"tabs\"`: Only draw tabs here.\n - `\"spaces\"`: Only draw spaces here.\n - `\"mixed\"`: Only draw white space that does not match the indentation style. For example if \"translate_tabs_to_spaces\" is true only draw tabs.\n - `\"mixed_tabs\"`: Like \"mixed\" but only draw tabs.\n - `\"mixed_spaces\"`: Like \"mixed\" but only draw spaces.\n - `\"all\"`: Draw both tabs and spaces. This is the default.\n\n Note that options are applied in sequence. So a later option may override an earlier one.\n\n**Examples:**\n\n- `[\"selection\", \"trailing\", \"isolated\"]`:\n   Draw white space at the end of any lines and under the selection.\n\n- `[\"all_tabs\", \"selection\"]`:\n   Draw tabs anywhere and any white space under the selection.\n\n- `[\"all_mixed\"]`:\n   Draw any white space that does not match the indentation style.\n\n- `[\"leading_mixed\", \"isolated_mixed\"]`:\n   Draw any indentation that does not match the indentation style.\n\n- `[\"selection_mixed_tabs\"]`:\n  Draw only tabs under the selection and only if the indentation style is spaces.\n\n- `[\"all_tabs\", \"selection\"]`:\n  Draw all tabs and any white space under the selection.\n\n- `[\"all\", \"selection_none\"]`:\n  Inverse of the default. Draw white space everywhere except under the selection.",
```

Comment:
I've removed some redundant newlines to fix rendering
```suggestion
          "markdownDescription": "Controls when white space is drawn.\n\nAny of the following options may be combined:\n\n - `\"selection\"`: Draw white space under the current selection.\n - `\"leading\"`: Draw any white space between the beginning of a line and the first character.\n - `\"enclosed\"`: Draw white space enclosed by other characters.\n - `\"trailing\"`: Draw white space following the last character on a line.\n - `\"isolated\"`: Draw white space on lines containing no other characters.\n - `\"all\"`: All of the above, ie. always draw white space.\n\nThese options may be further refined by appending any of the following separated by an underscore:\n\n - `\"none\"`: Don't draw this kind of white space.\n - `\"tabs\"`: Only draw tabs here.\n - `\"spaces\"`: Only draw spaces here.\n - `\"mixed\"`: Only draw white space that does not match the indentation style. For example if \"translate_tabs_to_spaces\" is true only draw tabs.\n - `\"mixed_tabs\"`: Like \"mixed\" but only draw tabs.\n - `\"mixed_spaces\"`: Like \"mixed\" but only draw spaces.\n - `\"all\"`: Draw both tabs and spaces. This is the default.\n\n Note that options are applied in sequence. So a later option may override an earlier one.\n\n**Examples:**\n\n- `[\"selection\", \"trailing\", \"isolated\"]`: Draw white space at the end of any lines and under the selection.\n- `[\"all_tabs\", \"selection\"]`: Draw tabs anywhere and any white space under the selection.\n- `[\"all_mixed\"]`: Draw any white space that does not match the indentation style.\n- `[\"leading_mixed\", \"isolated_mixed\"]`: Draw any indentation that does not match the indentation style.\n- `[\"selection_mixed_tabs\"]`: Draw only tabs under the selection and only if the indentation style is spaces.\n- `[\"all_tabs\", \"selection\"]`: Draw all tabs and any white space under the selection.\n- `[\"all\", \"selection_none\"]`: Inverse of the default. Draw white space everywhere except under the selection.",
```

![Screenshot 2020-09-19 at 20 05 41](https://user-images.githubusercontent.com/153197/93685998-ae627700-fab3-11ea-94e7-0a11b903cfb7.png)